site/docs/components/icons.md

```diff
@@ -36,25 +41,83 @@ Apply `.icon`, `.icon-sm`, and `.icon-lg` to size icons
 
 
 ### Color
-
-Add `.text-$color` classes to apply colors to icons and select logos
+Setting `fill=currentColor` gives control of coloring the SVG colors through CSS and added classes.
+Here you can see by adding `.text-$color` classes to it controls the changing the colors.
 
```

Comment:
Suggestion: "Setting `fill=currentColor` in the SVG allows the SVG's colors to be controlled by CSS and added classes. Below you can see the SVG's color changing by adding `.color-` classes to the svg tag."

Reply:
Dig it . Thanks